File: CSHARP/Cap03/c03ex03.cs
// c03ex03.cs

using System;
using static System.Console;

public class Program
{
    public static void Main(string[] args)
    {
        double HT, VH, PD, SB, TD, SL;

        Write("Entre quantidade de horas trabalhadas ..: ");
        HT = double.Parse(ReadLine());

        Write("Entre valor do salario-hora ............: ");
        VH = double.Parse(ReadLine());

        Write("Entre valor do percentual de desconto ..: ");
        PD = double.Parse(ReadLine());

        SB = HT * VH;
        TD = (PD / 100) * SB;
        SL = SB - TD;

        WriteLine("\nSalario Bruto ....: {0:F2}", SB);
        WriteLine("Salario Liquido ..: {0:F2}", SL);

        Write("\nAperte <Enter> para encerrar... ");
        ReadLine();
    }
}

File: DART/Cap03/c03ex03.dart
// c03ex03.dart

import 'dart:io';

void main() {
  double? ht;
  double? vh;
  double? pd;
  double sb = 0.0;
  double td = 0.0;
  double sl = 0.0;

  stdout.write("Entre quantidade de horas trabalhadas ..: ");
  ht = double.tryParse(stdin.readLineSync() ?? '');

  stdout.write("Entre valor do salario-hora ............: ");
  vh = double.tryParse(stdin.readLineSync() ?? '');

  stdout.write("Entre valor do percentual de desconto ..: ");
  pd = double.tryParse(stdin.readLineSync() ?? '');

  sb = (ht ?? 0.0) * (vh ?? 0.0);
  td = ((pd ?? 0.0) / 100) * sb;
  sl = sb - td;

  print("\nSalario Bruto ....: ${sb.toStringAsFixed(2)}");
  print("Salario Liquido ..: ${sl.toStringAsFixed(2)}");

  stdout.write("\nTecle <Enter> para encerrar... ");
  stdin.readLineSync();
}
